components/CardAuthentication/src/_mixin.scss

```diff
@@ -75,8 +86,19 @@
     margin-inline-end: auto;
     margin-block-end: var(--denhaag-card-authentication-portrait-logo-margin-block-end);
   }
+  .denhaag-card-authentication__title {
+    margin-block-end: 18px;
```

Comment:
I think we might want to turn this into a design token. So `--denhaag-card-authentication-title-margin-block-end` and put the `18px` as a value in the design token file.

Reply:
Done!